GetAllPeople/index.ts

```diff
@@ -0,0 +1,22 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+import { MyPerson } from "../Person/PersonInterface";
+import errorHandler from "../Util/errorHandling";
+import mongooseConnection from "../Util/mongooseConnection";
+
+const httpTrigger: AzureFunction = async function (
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  try {
+    await mongooseConnection();
+
+    context.res = {
+      body: await MyPerson.find({}),
```

Comment:
I also want you to use the interface of the person so it will be more clear
```suggestion
     const people: MyPerson[] = await MyPerson.find({})
```